### DISCUSSION TITLE
osgamejobs

### DISCUSSION THREAD

**Discussion by hhirsch** [opengaming/osgameclones#1383] [General]
Hello piranha,

we are always looking for new talents. What do you think about the site osgamejobs where
open source games can post their jobs. Can you open such a site?

Regards
Henry

**Comment by wonea** [comment]
Sounds interesting, does something like this already exist?

**Comment by hhirsch** [comment]
There used to be a site gamecollab.net which was down all of a sudden and never came back and there are forums like gamedev.net

**Comment by sansolovyov** [comment]
Oh damn, I thought I wrote this comment, but it was just sitting in a tab not yet submitted. Anyway, here it is:

Whole idea of the site was to help you discover new games _plus_ to recruit new people for development of those games. That's why we include some information about languages and technologies.

I'm all for including more data, like "this game is actively seeking for developers/artists", but how do we update it so it's not going to be completely stale? Mark with dates? Any thoughts?

**Comment by wonea** [comment]
I was thinking about this today. Perhaps the best way would be grab the latest changes from each repository, and set this against a set of criteria; changes in the last week - actively development, 1 change in the last year - sporadic, no changes in five years - halted development

Okay this would be a fair amount of work, and probably something that could be ran with Travis and updated with new commits. As a start shall we move game status into a separate field, this would save parsing the info field?

**Comment by sansolovyov** [comment]
Hmm... Having new commits happening for that is an interesting idea for sure. Ok, I like the idea, we probably need to convert our more or less free-form `info` field into something more structured, and then have a bot which goes through repos and determines activity. :-)

**Comment by cxong** [comment]
We can extract the date of the last commit in any branch, use http://timeago.yarp.com/ and let the user decide how active that project is.

**Comment by sansolovyov** [comment]
Can't decide how often should this be done. Daily seems good enough, but then that means amount of commits would be enormous... Maybe db should be moved to a different repo?

**Comment by cxong** [comment]
Dynamically generated data like this doesn't belong in source control, but project info should be, so there will be two dbs.
